boards/arm/nucleo_h743zi/pinmux.c

```diff
@@ -14,6 +14,37 @@
 
 /* NUCLEO-H743ZI pin configurations */
 static const struct pin_config pinconf[] = {
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(usart3), okay) && CONFIG_SERIAL
+	{ STM32_PIN_PD8, STM32H7_PINMUX_FUNC_PD8_USART3_TX },
+	{ STM32_PIN_PD9, STM32H7_PINMUX_FUNC_PD9_USART3_RX },
+#endif
```

Comment:
There are now configured through device tree with following props:
```
pinctrl-0 = <&usart3_tx_pd8 &usart3_rx_pd9>;
```
We're in the process of this migration.

Reply:
Good point, see my comments too for SPI node 😄